test/integration-test.py

```diff
@@ -140,6 +142,32 @@ def test_multidomain():
 def test_dns_challenge():
     auth_and_issue([random_domain(), random_domain()], chall_type="dns-01")
 
+def test_issuer():
+    """
+    Issue a certificate, fetch its chain, and verify the chain and
+    certificate against test/test-root.pem. Note: This test only handles chains
+    of length exactly 1.
+    """
+    certr, authzs = auth_and_issue([random_domain()])
+    cert = urllib2.urlopen(certr.uri).read()
+    chain = urllib2.urlopen(certr.cert_chain_uri).read()
```

Comment:
`urllib2.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:661)>`

I think verification needs to be disabled here (or the root store needs to be modified to contain the root here).